src/components/ButtonGroup.tsx

```diff
@@ -0,0 +1,53 @@
+import { SortType } from '../enums/sort-type.enum';
+import { Button } from './Button';
+
+type Props = {
+  sortField: SortType | '';
+  isReversed: boolean;
+  setSortField: (sortField: SortType | '') => void;
+  setIsReversed: (isReversed: boolean) => void;
+  reset: () => void;
+};
+
+export const ButtonGroup: React.FC<Props> = ({
+  sortField,
+  isReversed,
+  setSortField,
+  setIsReversed,
+  reset,
+}) => {
+  const handleSortAlphabetically = () => setSortField(SortType.ALPHABETICALLY);
+  const handleSortByLength = () => setSortField(SortType.BY_LENGTH);
+  const handleReverse = () => setIsReversed(!isReversed);
+  const handleReset = () => reset();
+
+  return (
+    <div className="buttons">
+      <Button
+        color="is-info"
+        onClick={handleSortAlphabetically}
+        isLight={sortField !== SortType.ALPHABETICALLY}
+      >
```

Comment:
[CODE KNOWLEDGE] - When rendering a list, don't forget to add `key` prop. Even though you are not rendering a list here, it's a good practice to have a `key` prop on components that could potentially be part of a list to avoid issues in the future.